### DISCUSSION TITLE
Guidance for the application process of GSOC'23

### DISCUSSION THREAD

**Discussion by Pratyusha132** [alan-turing-institute/sktime#4069] [General]
Hi everyone, I am currently a final year Computer Science undergraduate student at the Kalinga Institute of Industrial Technology, India. I am deep enthusiast for web development and machine learning and have worked on a number of projects in the past that dealt with these technologies. In addition to that I have a hands on MERN stack, Pytorch.
I want to contribute to sktime. If the organization is taking part in Google Summer of Code this year, I’m also interested in applying for the program too. Since I’m a complete novice to sktime, it would be great if the mentors can guide me as to how I may become more involved such as things I should study before digging into the code and issues that are appropriate for a beginner like me. I'm eager to pick up difficult projects and learn any new technology.

Thanking You,
Pratyusha Giri

**Comment by fkiraly** [comment]
welcome to sktime, @Pratyusha132!
The same question was asked, and answered here: https://github.com/sktime/sktime/discussions/4046
